### PR TITLE
LPS-144598 Activate One-to-One Relationship with Feature Flag

### PR DESCRIPTION
**Pull request by mateussgn**
Related issue: https://issues.liferay.com/browse/LPS-144598

### CODE REVIEW

**Comment by mateussgn** [comment]
ci:test:relevant

**Review by marcelabc — CHANGES_REQUESTED** [review]

File: modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/ObjectDefinitionsRelationshipsDisplayContext.java [hunk 1]
```diff
@@ -43,12 +44,17 @@
 	public ObjectDefinitionsRelationshipsDisplayContext(
 		HttpServletRequest httpServletRequest,
 		ModelResourcePermission<ObjectDefinition>
-			objectDefinitionModelResourcePermission) {
+			objectDefinitionModelResourcePermission,
+		FFOneToOneRelationshipConfigurationActivator
```
[the remark this answers is not in the document]
Comment (mateussgn) [reply modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/ObjectDefinitionsRelationshipsDisplayContext.java:48]: Sure! Doing right now!

**Comment by mateussgn** [comment]
ci:test:relevant

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by marcelabc** [comment]
ci:forward

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
Hey @mateussgn, could you check this SF failure please? Thanks! 😄

**Comment by mateussgn** [comment]
ci:test:sf

**Comment by mateussgn** [comment]
ci:test:sf

**Comment by mateussgn** [comment]
ci:forward

**Pull request closed without merging**